spec/components/contextual_breadcrumbs_spec.rb

```diff
@@ -42,13 +42,28 @@ def tag_to_brexit(content_item)
     content_item
   end
 
+  def tag_to_priority_coronavirus(content_item)
+    content_item["links"]["taxons"][1]["title"] = "Coronavirus Education"
+    content_item["links"]["taxons"][1]["content_id"] = "272308f4-05c8-4d0d-abc7-b7c2e3ccd249"
+    content_item
+  end
+
   it "renders the brexit-superbreadcrumb meta tag on content tagged to brexit" do
     content_item = example_document_for("guide", "guide")
     content_item = tag_to_brexit(content_item)
     render_component(content_item: content_item)
     assert_select "meta[name='govuk:brexit-superbreadcrumb'][content='Brexitbusiness']"
   end
 
+  it "does not render the brexit-superbreadcrumb meta tag on content tagged to brexit AND a priority coronavirus taxon" do
+    content_item = example_document_for("guide", "guide")
+    content_item = tag_to_brexit(content_item)
+    content_item["links"]["taxons"] << {}
+    content_item = tag_to_priority_coronavirus(content_item)
+    render_component(content_item: content_item)
+    assert_select "meta[name='govuk:brexit-superbreadcrumb'][content='Brexitbusiness']", count: 0
```

Comment:
I got a bit confused here until I realised they were two different metatags 🤦

Reply:
Yeah it IS confusing! Thank you for the review 😅